### PR TITLE
Support Delete resource

### PR DESCRIPTION
**Pull request by runzexia**

### CODE REVIEW

**Comment by runzexia** [comment]
@gavinfish

File: src/main/resources/com/microsoft/jenkins/kubernetes/KubernetesDeployContext/config.properties [hunk 1]
```diff
@@ -18,3 +18,5 @@ enableConfigSubstitution_title = Enable Variable Substitution in Config
 dockerCredentialsSection_title = Docker Container Registry Credentials / Kubernetes Secrets
 secretName_title = Secret Name
 dockerCredentials_title = Docker Container Registry Credentials
+
+deleteResource_title = Delete Resources.
```
Comment (gavinfish) [inline src/main/resources/com/microsoft/jenkins/kubernetes/KubernetesDeployContext/config.properties]: Full stop is unnecessary here.

Comment (runzexia) [reply src/main/resources/com/microsoft/jenkins/kubernetes/KubernetesDeployContext/config.properties]: I don't understand what it means. Can you explain it more?

Comment (gavinfish) [reply src/main/resources/com/microsoft/jenkins/kubernetes/KubernetesDeployContext/config.properties]: We don't need the `.` at the end 😄 .

Comment (runzexia) [reply src/main/resources/com/microsoft/jenkins/kubernetes/KubernetesDeployContext/config.properties]: fix

**Pull request merged**